packages/driver/cypress/integration/commands/clock_spec.js

```diff
@@ -115,6 +115,8 @@ describe('src/cy/commands/clock', () => {
 
     // this test was written to catch a bug in lolex (dep, now @sinonjs/fake-timers) 3 and was fixed by lolex 4 upgrade,
     it('doesn\'t override window.performance members', () => {
+      cy.visit('/fixtures/empty.html')
+
```

Comment:
Why are we visiting here when we weren't before? I see a few places in the driver specs where `.visit`s were added where there were none before.

Reply:
I think this is because we've enable the experimental flag for all driver tests, which enables session support, which requires visiting in every test since it resets to `about:blank` between tests.

Reply:
Yeah, I was gonna comment on that, I think it's a bad idea to enable the experimental flag for all driver tests. We don't do this for any other experimental flag, because it means that all of our other driver tests now run with the assumption that an "experimental" flag is enabled.

I see two ways of fixing this...

1. Synchronize the driver's value of `experimentalSessionAndOrigin` with the `proxy` at some point so that it can be updated before tests
2. Move the tests that require `experimentalSessionAndOrigin` to be set in the `proxy` to a `system-tests` project so they run separately from the driver tests

Thoughts @chrisbreiding ?

Reply:
I think we'd have to go with option 1, since option 2 would mean moving _all_ of the cross-origin driver tests to system-tests, which would be terrible for iterating on them. I'll look into updating the flag in the proxy in real-time, so that it doesn't need to be enabled for all the tests.

Reply:
A third option be setup a second cypress configuration file in the driver package and remove the need for an additional proxy and create a new script for running these tests. This would be closer to how users would be enabling this flag.

Reply:
We discussed this and decided we're going to leave things as-is for the time being, but work on a better solution once the experimental release is out. It will probably be something like option 1 that allows syncing experimental flags with the server.

Reply:
#21148 opened by @mjhenkes to address the concerns here in the interim until we can sync test overrides with the server, or something of the sort.